packages/react-dom/src/__tests__/ReactDOM-test.js

```diff
@@ -366,7 +385,7 @@ describe('ReactDOM', () => {
     }
   });
 
-  it('should not crash with devtools installed', () => {
+  it('should not crash with devtools installed', async () => {
```

Comment:
This test is weird because there's no assertions?

Reply:
I guess it's just "Should not crash", ie. an implicit "assert(..).not.toThrow()".